app/controllers/quotes_controller.rb

```diff
@@ -16,15 +16,20 @@ def index
 
   def create
     authenticate_user!
-    @anime = Anime.find(params["quote"]["anime_id"])
-    @quote = Quote.new
-    @quote.content = params["quote"]["content"]
-    @quote.character_name = params["quote"]["character_name"]
-    @quote.anime = @anime
-    @quote.user = current_user
-    @quote.save
-
-    redirect_to anime_quotes_path(@anime)
+
+    quote_params = params.require(:quote).permit(
+      :anime_id, :character_name, :content, :favorite_count, :username
```

Comment:
Uh.  Should the user really be able to set the `favorite_count`? And should `username` be in here too?  I think you might be abusing `permit`.